NEXTCLOUD.cmake

```diff
@@ -6,7 +6,8 @@ set( APPLICATION_VENDOR     "Nextcloud GmbH" )
 set( APPLICATION_UPDATE_URL "https://updates.nextcloud.org/client/" CACHE STRING "URL for updater" )
 set( APPLICATION_HELP_URL   "" CACHE STRING "URL for the help menu" )
 set( APPLICATION_ICON_NAME  "Nextcloud" )
-set( APPLICATION_SERVER_URL "" CACHE STRING "URL for the server to use. If entered the server can only connect to this instance" )
+set( APPLICATION_SERVER_URL "" CACHE STRING "URL for the server to use. If entered, the UI field will be pre-filled with it" )
+option( APPLICATION_SERVER_URL_ENFORCE "Enforce pre-defined server URL" ON ) # If set and APPLICATION_SERVER_URL is defined, the server can only connect to the pre-defined URL
```

Comment:
I'd expect a set here, otherwise since we include that file it'd mean we declare the option twice. I'm pretty sure cmake will try to punish us for this at some point.

Reply:
Ah yeah, I also like this more for readability. Now with `set` it is also working without the additional change in `CMakeLists.txt`.